README.md

```diff
@@ -16,14 +16,14 @@ To play, you require *an original AoE II: TC or [AoE II: HD](http://store.steamp
 
 Contact
 -------
-Contact  | Where?
----------|-------
-Issue Tracker | [SFTtech/openage](https://github.com/SFTtech/openage/issues)
+Contact          | Where?
+-----------------|-------
+Issue Tracker    | [SFTtech/openage](https://github.com/SFTtech/openage/issues)
 Development Blog | [blog.openage.sft.mx](https://blog.openage.sft.mx)
-Forum | [<img src="https://www.redditstatic.com/about/assets/reddit-logo.png" alt="reddit" height="22"/> /r/openage](https://www.reddit.com/r/openage/)
-Matrix Chat | [`#sfttech:matrix.org`](https://riot.im/app/#/room/#sfttech:matrix.org)
-IRC Chat | [`irc.freenode.net #sfttech`](https://webchat.freenode.net/?channels=sfttech)
-Money Sink | [![money sink](https://liberapay.com/assets/widgets/donate.svg)](https://liberapay.com/SFTtech)
+Forum            | [<img src="https://www.redditstatic.com/about/assets/reddit-logo.png" alt="reddit" height="22"/> /r/openage](https://www.reddit.com/r/openage/)
+Matrix Chat      | [`#sfttech:matrix.org`](https://riot.im/app/#/room/#sfttech:matrix.org)
+IRC Chat         | [`irc.freenode.net #sfttech`](https://webchat.freenode.net/?channels=sfttech)
+Money Sink       | [![money sink](https://liberapay.com/assets/widgets/donate.svg)](https://liberapay.com/SFTtech)
```

Comment:
you could change the blog to `blog.openage.dev`.
the spacing around the `|` was chosen like this deliberately in order to not having to reindent all lines once we change one of them :) but it doesn't hurt to align them, dunno.

Reply:
Done